source/frontend/package-lock.json

```diff
@@ -16027,6 +16027,12 @@
         "@types/yargs-parser": "*"
       }
     },
+    "node_modules/jquery": {
+      "version": "3.6.3",
+      "resolved": "https://registry.npmjs.org/jquery/-/jquery-3.6.3.tgz",
+      "integrity": "sha512-bZ5Sy3YzKo9Fyc8wH2iIQK4JImJ6R0GWI9kL1/k7Z91ZBNgkRXE6U0JfHIizZbort8ZunhSI3jw9I6253ahKfg==",
+      "peer": true
+    },
```

Comment:
hmm weird to see package.lock updated without corresponding change in package.json

could you verify this change is needed?

Reply:
My guess it was not updated before and after running the install it updated the lock file as well